suite-common/wallet-utils/src/sendFormUtils.ts

```diff
@@ -719,3 +719,97 @@ export const getMevProtectedTxData = (
 export const isExchangeTradingForm = (
     form: FormStateTrading | undefined,
 ): form is FormStateTradingExchange => form?.activeSection === 'exchange';
+
+interface GetNetworkReserveProps {
+    symbol: NetworkSymbol;
+    contractAddress: string | undefined | null;
+    isEnabled?: boolean;
+}
+
+/**
+ * Reserve defined in networksConfig.ts applies to the native token only
+ */
+export const getNetworkReserve = ({
+    symbol,
+    contractAddress,
+    isEnabled,
+}: GetNetworkReserveProps) => {
+    if (
+        (!!contractAddress && contractAddress !== '0x0000000000000000000000000000000000000000') ||
+        !isEnabled
+    )
+        return undefined;
+    const network = getNetwork(symbol);
+
+    return network.nativeTokenReserve;
+};
+
+interface GetCryptoAmountWithReserveProps {
+    symbol: NetworkSymbol;
+    contractAddress: string | undefined | null;
```

Comment:
```suggestion
    contractAddress?: string | null;
```
then you don't need to set it to `undefined`

Reply:
done